util/cosmos.js

```diff
@@ -10,6 +10,9 @@ const {
   COSMOS_BLOCK_TIME = 5000,
 } = config;
 
+export const CONFIRMATION_NEEDED = 1;
```

Comment:
not imported / used anywhere?

Reply:
not yet used but I feel like exporting this would be useful later && to be consistent with exporting BLOCK related const